File: Programmers/src/Javascript/가장큰수찾기.js
function solution(array) {
    var answer = [];
    
    let max = 0;
    for(let i=0; i<array.length; i++){
        max = Math.max(max, array[i]);
    }
    let index = array.indexOf(max);
    
    answer.push(max);
    answer.push(index);
    
    return answer;
}

File: Programmers/src/Java/Lv0/가장 큰 수 찾기.java
package Java.Lv0;

import java.util.*;

class Solution {
    public int[] solution(int[] array) {
        int[] answer = new int[2];
        int max = 0;
        
        for(int num : array)
            max = Math.max(max, num);
        
        answer[0] = max;
        answer[1] = Arrays.binarySearch(array, max);
        return answer;
    }
}